proxy-attestation-server/src/cli.rs

```diff
@@ -0,0 +1,118 @@
+//! Proxy Attestation Server command-line interface
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use actix_rt;
+use env_logger;
+use log::{info, error};
+use proxy_attestation_server;
+use std::{env, fs, path, process};
+use structopt::StructOpt;
+use veracruz_utils::policy::{
+    error::PolicyError,
+    policy::Policy,
+};
+
+
+#[derive(Debug, StructOpt)]
+#[structopt(rename_all="kebab")]
```

Comment:
kebab?

Reply:
Kebab case, like "foo-bar-goo" instead of snake case "foo_bar_goo".

Reply:
Yep, https://docs.rs/structopt/0.3.22/structopt/#specifying-argument-types

I'm surprised kebab-case isn't the default, since it's usually the standard for CLIs `blah --goes-wee` vs `blah --goes_wee`

Reply:
Oh. Cool. Now I get it.